tests/test_submdspan_static_slice.cpp

```diff
@@ -567,7 +567,7 @@ TEST(TestMdspan, SubmdspanStaticSlice_Left_idd_FullTuple) {
   // in order for the output's extents type to be able to encode
   // the compile-time slice information.
   static_assert(std::is_convertible<std::tuple<IC<std::size_t, 1>, IC<std::size_t, 3>>,
-                                    std::tuple<std::size_t, std::size_t>>::value);
+                                    std::tuple<std::size_t, std::size_t>>::value, "convertability of tuples of integral constant to size_t is false");
```

Comment:
Oh oof, I missed that one! : - )  Thanks!!!

Reply:
its allowed in C++17

Reply:
typo convertability -> convertibility